.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav.vtl

```diff
@@ -0,0 +1,812 @@
+#*
+ * OmniNav v2 format.
+ * Chapman.edu/_components/omni_nav_v2/omni_nav.vtl
+ *
+ * It would make sense to move the various component-specific macros (like those
+ * for Primary Nav) into their own file and import them like so:
+ *
+ * #import( "/_components/omni_nav_v2/macros/primary_nav.vm" )
+ *
+ * But this makes debugging the code in Cascade's format editor much harder. It is
+ * simpler to just keep it all in a well-structured single file.
+ *#
+
+##
+## Hardcoded Params
+##
+## umbrellaCategories
+## These are locations for which OmniNav should build the secondary off-canvas-nav menu.
+## umbrellaCategory is the term product owners use.
+## Does order matter here? I'm assuming it does.
+## TODO: Talk with product owners to see if we can unify menu and brand labels [1] and [4] below.
+## Inner Array: [ $umbrellaDirectory, $umbrellaLabel/$umbrellaBrandLabel, $umbrellaClass, $umbrellaHref ]
+#set ( $umbrellaCategories = [
+  [ 'business', 'ARGYROS SCHOOL', 'asbe', '/business/' ],
+  [ 'education', 'ATTALLAH COLLEGE', 'education', '/education/' ],
+  [ 'dodge', 'DODGE COLLEGE', 'dodge', '/dodge/' ],
+  [ 'crean', 'CREAN COLLEGE', 'crean', '/crean/' ],
+  [ 'wilkinson', 'WILKINSON COLLEGE', 'wilkinson', '/wilkinson/' ],
+  [ 'copa', 'CoPA', 'copa', '/copa/' ],
+  [ 'pharmacy', 'CUSP', 'cusp', '/pharmacy/' ],
+  [ 'law', 'FOWLER SCHOOL OF LAW', 'fowler', '/law/' ],
+  [ 'scst', 'SCHMID COLLEGE', 'schmid', '/scst/' ],
+  [ 'about', 'About', 'default', '/' ],
+  [ 'academics', 'Academics', 'default', '/' ],
+  [ 'admission', 'Admission', 'default', '/' ],
+  [ 'alumni', 'Alumni', 'default', '/' ],
+  [ 'campus-services', 'Campus Services', 'default', '/' ],
+  [ 'faculty-staff', 'Faculty and Staff', 'default', '/' ],
+  [ 'families', 'Families', 'default', '/' ],
+  [ 'research', 'Research', 'default', '/' ],
+  [ 'students', 'Students', 'default', '/' ],
+  [ 'support-chapman', 'Support Chapman', 'default', '/' ],
+  [ 'communication', 'SCHOOL OF COMM', 'soc', '/communication/' ]
```

Comment:
I'm still waiting for Ross to get back to me on this list. He's reviewing the short names (2nd item in each list).